<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThuWndCircularDirectionKey</class>
 <widget class="QWidget" name="ThuWndCircularDirectionKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>241</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>241</width>
    <height>241</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>241</width>
    <height>241</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ThuWndCircularDirectionKey</string>
  </property>
  <widget class="QPushButton" name="id_btnTop">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>0</y>
     <width>157</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>157</width>
     <height>55</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btnBottom">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>185</y>
     <width>157</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>157</width>
     <height>56</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btnLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>44</y>
     <width>58</width>
     <height>153</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>58</width>
     <height>153</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btnRight">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>44</y>
     <width>58</width>
     <height>153</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>58</width>
     <height>153</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btnClockwiseRotation">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>62</y>
     <width>124</width>
     <height>57</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>124</width>
     <height>57</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btnReversal">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>122</y>
     <width>124</width>
     <height>57</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>124</width>
     <height>57</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
